### PR TITLE
NOISSUE - Export config

### PR DESCRIPTION
**Pull request by mteodor**
Fix export config saving when bootstraping

### CODE REVIEW

File: src/app/common/interfaces/bootstrap.interface.ts [hunk 1]
```diff
@@ -14,7 +14,38 @@ export interface ConfigContent {
   mqtt_url: string;
   edgex_url: string;
   nats_url: string;
-  export_config?: any;
+  export_config: ExportConfig;
+}
+
+export interface ExportConfig {
+  File: string;
```
Comment (manuio) [inline src/app/common/interfaces/bootstrap.interface.ts]: why upercase here?

File: src/app/pages/things/gateways/details/config/gateways.config.component.ts [hunk 2]
```diff
@@ -19,16 +19,37 @@ export class GatewaysConfigComponent implements OnInit, OnChanges {
     mqtt_url: '',
     edgex_url: '',
     nats_url: '',
-    export_config: '',
+    export_config: {
```
Comment (manuio) [inline src/app/pages/things/gateways/details/config/gateways.config.component.ts]: I see many values not initialized. Please, make them optional in the interface and remove them from here.

Comment (mteodor) [reply src/app/pages/things/gateways/details/config/gateways.config.component.ts]: what interface?

Comment (manuio) [reply src/app/pages/things/gateways/details/config/gateways.config.component.ts]: `MqttConfig`

File: src/app/common/services/bootstrap/bootstrap.service.ts [hunk 3]
```diff
@@ -19,41 +19,12 @@ export class BootstrapService {
...
-          mqtt_topic: '',
-          nats_topic: 'export.telegraf',
-          subtopic: '',
-          type: 'plain',
-        },
-      ],
-    },
+    export_config:  {
+      file:`${environment.exportConfigFile}`,
+      exp: {port:"8170",},
```
Comment (manuio) [inline src/app/common/services/bootstrap/bootstrap.service.ts]: you could add some spaces and tabulation here

**Pull request merged**